migrations/versions/0371_replace_template_content.py

```diff
@@ -0,0 +1,32 @@
+"""
+Revision ID: 371_replace_template_content
+Revises: 0370_notification_callback_url
+Create Date: 2024-10-08 20:35:38
+"""
+from alembic import op
+
+revision = '371_replace_template_content'
+down_revision = '0370_notification_callback_url'
+
+def upgrade():
+    op.execute("""
+        UPDATE templates
+        SET content = REPLACE(content, 'https://notification.alpha.canada.ca', 'https://api.va.gov/vanotify')
```

Comment:
This should probably be `https://notifications.va.gov` - please check with Melanie or Ksenia.

Reply:
Based on the content of the templates Mack demonstrated in her PR description, it makes sense to me that we should not provide any URL.  In the template, it looks like a URL for contacting the Notify team, and that's certainly not the API endpoint.  If we have to include a URL, I would link to the API repo on Github.

Reply:
We decided to completely remove the URL - this will be an irreversible change. 

https://oddball.slack.com/archives/C034496E4AU/p1728494887142629?thread_ts=1728484826.161349&cid=C034496E4AU